### ISSUE TITLE
"Column Definition" syntax is probably inaccurate and contradicts examples

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#289]
```
It looks like ``{ DEFAULT expression | AS computedColumnExpression }'' is in 
fact optional.
```

Original issue reported on code.google.com by `fercer...@gmail.com` on 23 Feb 2011 at 9:30

**Comment by GoogleCodeExporter** [comment]
```
Yes, of course it should be optional...
```

Original comment by `thomas.t...@gmail.com` on 24 Feb 2011 at 7:24
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.152.
```

Original comment by `thomas.t...@gmail.com` on 1 Mar 2011 at 8:05
- Changed state: **Fixed**
